src/commands/watcher.ts

```
@@ -22,22 +22,22 @@ export interface DaemonInstallOptions {
   nodePath?: string;
 }
 
-export interface DaemonUninstallOptions {
+export interface WatcherUninstallOptions {
   yes?: boolean;
 }
 
-export interface DaemonStartOptions {}
+export interface WatcherStartOptions {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)